content/blog/2020/2020-04-09-covid-19-part-2.md

```diff
@@ -0,0 +1,160 @@
+
+{{% tooltippic src="/blog_img/2020/2020-04-09-edit-domain.gif" %}}
+
+The data is now ready to be processed by Time Slice. 
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-05.png" %}}
+
+{{% tooltippic src="/blog_img/2020/2020-04-09-time-slice.png" %}}
+
+
+The top indicates the currently selected slice of data. You can change the interval by interacting with the histogram, or by adjusting the start and end dates directly below. Let's select a day's slice between the 25th and 26th January 2020. 
+
+With a click of the play button, the slice moves, and the data output updates accordingly. To verify that it works, connect a Data Table, and see it in action.
+
+
+### Kill Transpose: Volume 2
+
+Make sure to stop the Time Slice playback before proceeding. We're almost at the finish line; we just need to transform the data back into its original form. To do so, connect another Transpose widget.
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-06.png" %}}
+
+Under *Future names* select *Generic*, which will name the new column *Feature 1*.
+
+{{% tooltippic src="/blog_img/2020/2020-04-09-transpose-2.png" %}}
+
+Let's connect a Data Table for a final check of our data before putting it on a map.
+
+{{% figure src="/blog_img/2020/2020-04-09-table-2.png" %}}
+
+Looking good!
+
+The final step entails connecting the Choropleth Map, selecting *Feature 1* as *Attribute*, and *Sum* as the *Agg*.
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-07.png" %}}
+
+To observe the animation, open up both Choropleth and Time Slice at the same time.<br> 
+Protip: pressing Ctrl/CMD+Up reveals the workflow, Ctrl/CMD+Down reveals widget windows. 
+
+All that's left is to hit play.
+
+{{% video src="/blog_img/2020/2020-04-09-animation-choropleth.mp4" %}}
+
+Now that we're familiar with Time Slice, let's use it with Geo Map too. For bonus points, let's also remove countries with zero confirmed cases. That way, the number of points on the map will increase as the virus spreads throughout the world. For this, we'll use a Select Rows widget.
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-08.png" %}}
+
+Under Conditions, select *Feature 1*, select *is greater than*, and type in *0*.
+
+{{% tooltippic src="/blog_img/2020/2020-04-09-select-rows.gif" %}}
+
+Add and connect a Geo Map widget. Zoom out, so you see the whole world, and on the left side, check *Freeze map*. This will prevented the map from jumping around as new points are added.
+
+{{% sidework src="/blog_img/2020/2020-04-09-workflow-09.png" %}}
+
+Press play once again – let's see what we've built!
+
+{{% video src="/blog_img/2020/2020-04-09-animation-geomap.mp4" %}}
+
+## Where from here?
+
+We learned how to extend Orange's functionality with new widgets from add-ons. We used the Orange3-Geo add-on to put COVID-19's numbers on the map. Also, we created an animation showing the spread of the virus. This change could be a good indicator of how countries are fighting the epidemic. 
```

Comment:
The conclusion summarizes the content, but doesn't offer any insights/synthesis. A couple of sentences should be added here.

'This change could be a good indicator of how countries are fighting the epidemic. ' is a step in this direction, but I don't understand what the 'this' in the sentence is referring to.